### PR TITLE
out_s3: fix document

### PR DESCRIPTION
**Pull request by jedipunkz**
I fix document of s3` output plugin.
Sorry, I re-create PR

----
Enter `[N/A]` in the box, if an item is not applicable to your change.

**Testing**
Before we can approve your change; please submit the following in a comment:
- [N/A] Example configuration file for the change
- [ N/A Debug log output from testing the change

- [N/A] Attached [Valgrind](https://valgrind.org/docs/manual/quick-start.html) output that shows no leaks or memory corruption was found

**Documentation**

- [x] Documentation required for this feature

----

Fluent Bit is licensed under Apache 2.0, by submitting this pull request I understand that this code will be released under the terms of that license.

### CODE REVIEW

**Review by PettitWesley — APPROVED** [review]

**Comment by PettitWesley** [comment]
The message is better, though a more descriptive message would be ideal. May be "out_s3: fix documentation for total_file_size option".

Also, when you commit the message you need to use signoff, like `git commit --sign`

**Comment by PettitWesley** [comment]
Without DCO (`--sign`) on the commit message, we can't merge this.

**Comment by jedipunkz** [comment]
sorry, I re-create PR https://github.com/fluent/fluent-bit/pull/3647 . and I close this PR.

**Pull request closed without merging**
